nengo_spa/algebras/tests/test_algebras.py

```diff
@@ -147,6 +195,33 @@ def test_identity_element(algebra, sidedness, rng):
             assert np.allclose(algebra.bind(a, p), a)
 
 
+@pytest.mark.parametrize("sidedness", ElementSidedness)
+def test_negative_identity_element(algebra, sidedness, rng):
+    try:
+        x = next(UnitLengthVectors(16, rng=rng))
+        a = algebra.abs(x)
+        with warnings.catch_warnings(record=True) as caught_warnings:
+            warnings.simplefilter("always", DeprecationWarning)
+            p = algebra.negative_identity_element(16)
+    except NotImplementedError:
+        pass
```

Comment:
Not sure I'm the biggest fan of these `except NotImplementedError -> pass` clauses. I think `pytest.skip()` would be a bit better. The rationale being if someone is running individual tests to debug things, or to develop their own custom algebra, then there's no way to distinguish between a test passing because it was implemented correctly versus wasn't implemented at all.

Reply:
That's a good point and I'll consider it. But I think skipped tests are listed in the summary statistics of pytest and it might be unclear whether the tests were skipped because some flag/dependency/etc. was missing or because they are always skipped because an algebra does not implement in the feature. Maybe `xfail` would be more appropriate (because an algebra not implementing the feature is expected to fail the test), but not entirely sure. I feel it is still a slightly different usage than what `xfail` is usually used for.